variables.tf

```diff
@@ -82,6 +82,17 @@ variable "nat_instance_type" {
   default     = "t3.micro"
 }
 
+variable "nat_instance_cpu_credits_override" {
+  type        = string
+  description = "NAT Instance credit option for CPU usage. Valid values include standard or unlimited. T3 instances are launched as unlimited by default. T2 instances are launched as standard by default."
+  default     = ""
+
+  validation {
+    condition     = contains(["standard", "unlimited"], var.nat_instance_cpu_credits_override)
```

Comment:
@bradj This validation is failing because the default is neither of these. Mind accounting for that?